csrc/ir/nodes.cpp

```diff
@@ -4996,15 +4996,21 @@ _scaled_dot_product_flash_attention_meta(const at::Tensor& query) {
   int seqlen_q = sizes[2];
   auto logsumexp = at::empty(
       {batch_size, num_heads, seqlen_q}, query.options().dtype(at::kFloat));
+  // Produce defined meta tensors for philox outputs so downstream segments
+  // can bind metadata and types correctly.
+  const auto meta_u64 =
+      at::TensorOptions().device(at::kMeta).dtype(at::kUInt64);
+  auto rng_state = at::empty({2}, meta_u64); // philox_seed/rng_state
```

Comment:
The comment is a a bit confusing. rng_state is a combination of seed and itself?

Reply:
Good question. The confusion comes from here:
https://github.com/pytorch/pytorch/blob/cdc8460f2c76f98ba30556e3f9358e857a2f22f0/aten/src/ATen/native/transformers/cuda/flash_attn/flash_api.cpp#L773-L778
I will add a comment to the code pointer towards this link.

Reply:
I see pytorch requirement > 2.7.1 on Thunder.
https://github.com/Lightning-AI/lightning-thunder/blob/ed4299f99ffd6ba74111816838c2779db4ddbfff/requirements/base.txt#L1
I'll check with Thunder if they are okay with removing previous behavior since they are not testing this.